docs/quickstart.rst

```diff
@@ -143,7 +143,7 @@ To start Solr using a volume mount and expose the admin webapp on port 8983:
 
   export SOLR_DATA=~/nexus-quickstart/solr
   mkdir -p ${SOLR_DATA}
-  docker run --name solr --network sdap-net -v ${SOLR_DATA}/:/bitnami -p 8983:8983 -e SDAP_ZK_SERVICE_HOST="host.docker.internal" -d ${REPO}/sdap-solr-cloud:${SOLR_VERSION}
+  docker run --name solr --network sdap-net -v ${SOLR_DATA}/:/bitnami -p 8983:8983 -e ZK_HOST="host.docker.internal:2181/solr" -d ${REPO}/sdap-solr-cloud:${SOLR_VERSION}
```

Comment:
The exact opposite of this change was made by Stepheny [several months ago](https://github.com/apache/sdap-nexus/commit/4a386a53ec0390a45c77f2032fe2ce87ca7d7525#diff-c9a9d6773bfb8f6a209256e8f1dc00db3eca97be188d9541607f55ca4f9744d1L146-R146):

This tells me we need to figure out why one configuration works for some users and the other works for others rather than just using one.

Reply:
I don't see how setting SDAP_ZK_SERVICE_HOST has any effect.
The Solr image is based of this Dockerfile https://github.com/apache/sdap-nexus/blob/develop/docker/solr/Dockerfile which is based off bitnami/solr:8.11.1-debian-10-r0, so how would it make use of an SDAP specific environment variable?

Reply:
I'm a bit confused too
But the discussion in #298 showed it worked for both Stepheny and a user on dev@

Reply:
After additional investigations with Riley, we decided to update the env to match what is used for the bitnami solr chart at https://github.com/bitnami/charts/blob/6db8a2e736e2a60be2d3848d85f49d236a43136c/bitnami/solr/templates/statefulset.yaml#L239-L240